Thanks! I got the following compilation errors when I tried to compile this PR.

```
---- Build started: Project: SeeGitApp, Configuration: Debug Any CPU ------
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2081,5): warning MSB3245: Could not resolve this reference. Could not locate the assembly "Microsoft.Expression.Interactions". Check to make sure the assembly exists on disk. If this reference is required by your code, you may get compilation errors.
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2081,5): warning MSB3245: Could not resolve this reference. Could not locate the assembly "System.Windows.Interactivity". Check to make sure the assembly exists on disk. If this reference is required by your code, you may get compilation errors.
1>Z:\dev\haacked\seegit\SeeGitApp\Views\CommitVertexView.xaml(23,10): error MC3074: The tag 'Interaction.Behaviors' does not exist in XML namespace 'http://schemas.microsoft.com/expression/2010/interactivity'. Line 23 Position 10.
2>------ Build started: Project: UnitTests, Configuration: Debug Any CPU ------
2>CSC : error CS0006: Metadata file 'Z:\dev\haacked\seegit\SeeGitApp\bin\Debug\SeeGit.exe' could not be found
========== Build: 0 succeeded, 2 failed, 0 up-to-date, 0 skipped ==========
```

--- a/SeeGitApp/SeeGitApp.csproj
+++ b/SeeGitApp/SeeGitApp.csproj
@@ -44,11 +44,11 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="GraphShape, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\GraphShape.1.0.0\lib\net40\GraphShape.dll</HintPath>
+    <Reference Include="GraphShape, Version=1.1.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\GraphShape.1.1.0\lib\net40\GraphShape.dll</HintPath>
     </Reference>
-    <Reference Include="GraphShape.Controls, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\GraphShape.Controls.1.0.0\lib\net40\GraphShape.Controls.dll</HintPath>
+    <Reference Include="GraphShape.Controls, Version=1.1.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\GraphShape.Controls.1.1.0\lib\net40\GraphShape.Controls.dll</HintPath>
     </Reference>
     <Reference Include="LibGit2Sharp, Version=0.24.0.0, Culture=neutral, PublicKeyToken=7cbde695407f0333, processorArchitecture=MSIL">
       <HintPath>..\packages\LibGit2Sharp.0.24.0\lib\net40\LibGit2Sharp.dll</HintPath>
@@ -63,8 +63,8 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\packages\Windows7APICodePack-Shell.1.1.0.0\lib\Microsoft.WindowsAPICodePack.Shell.dll</HintPath>
     </Reference>
-    <Reference Include="QuikGraph, Version=2.1.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\QuikGraph.2.1.0\lib\net40\QuikGraph.dll</HintPath>
+    <Reference Include="QuikGraph, Version=2.2.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\QuikGraph.2.2.0\lib\net40\QuikGraph.dll</HintPath>
     </Reference>
     <Reference Include="System" />
     <Reference Include="System.configuration" />

--- a/SeeGitApp/packages.config
+++ b/SeeGitApp/packages.config
@@ -1,10 +1,10 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="GraphShape" version="1.0.0" targetFramework="net40" />
-  <package id="GraphShape.Controls" version="1.0.0" targetFramework="net40" />
+  <package id="GraphShape" version="1.1.0" targetFramework="net40" />
+  <package id="GraphShape.Controls" version="1.1.0" targetFramework="net40" />
   <package id="LibGit2Sharp" version="0.24.0" targetFramework="net462" />
   <package id="LibGit2Sharp.NativeBinaries" version="1.0.185" targetFramework="net462" />
-  <package id="QuikGraph" version="2.1.0" targetFramework="net40" />
+  <package id="QuikGraph" version="2.2.0" targetFramework="net40" />
   <package id="Rx_Experimental-Main" version="1.1.11111" requireReinstallation="true" />
   <package id="Windows7APICodePack-Core" version="1.1.0.0" targetFramework="net40-Client" />
   <package id="Windows7APICodePack-Shell" version="1.1.0.0" targetFramework="net40-Client" />

--- a/UnitTests/UnitTests.csproj
+++ b/UnitTests/UnitTests.csproj
@@ -49,8 +49,8 @@
       <HintPath>..\packages\Moq.4.8.1\lib\net45\Moq.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="QuikGraph, Version=2.1.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\QuikGraph.2.1.0\lib\net40\QuikGraph.dll</HintPath>
+    <Reference Include="QuikGraph, Version=2.2.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\QuikGraph.2.2.0\lib\net40\QuikGraph.dll</HintPath>
     </Reference>
     <Reference Include="System" />
     <Reference Include="System.Configuration" />

--- a/UnitTests/packages.config
+++ b/UnitTests/packages.config
@@ -2,7 +2,7 @@
 <packages>
   <package id="Castle.Core" version="4.2.1" targetFramework="net46" />
   <package id="Moq" version="4.8.1" targetFramework="net46" />
-  <package id="QuikGraph" version="2.1.0" targetFramework="net40" />
+  <package id="QuikGraph" version="2.2.0" targetFramework="net40" />
   <package id="System.Threading.Tasks.Extensions" version="4.4.0" targetFramework="net46" />
   <package id="System.ValueTuple" version="4.4.0" targetFramework="net46" requireReinstallation="true" />
   <package id="WPFExtensions" version="1.0.0" targetFramework="net40" />

